scripts/export-package.sh

```diff
@@ -18,37 +18,45 @@
 # │       ├── bugsnag-plugin-android-ndk.aar
 # │       └── bugsnag-plugin-android-anr.aar
 # ├── cocoa
-# │   ├── libbugsnag-cocos2dx-ios.a
-# │   └── libbugsnag-cocos2dx-macosx.a
+# │   └── BugsnagCocos2dx.xcframework
 # └── include
 #     └── BugsnagCocos2dx
 #         ├── cocoa/{bugsnag-cocoa public headers}
 #         ├── cocoa/BugsnagCocos2dxPlugin.h
 #         └── Bugsnag.hpp
 
 PROJ_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")"/.. && pwd)"
+BUILD_DIR="$PROJ_DIR/build"
+PKG_DIR="$BUILD_DIR/pkg/bugsnag"
```

Comment:
This is a different package dir than was used before, and it creates a new intermediate directory named `bugsnag`, which could potentially break automated setups if upgraded from an existing installation.

```suggestion
PKG_DIR="$BUILD_DIR/pkg"
```

Reply:
It is working as expected, but the naming is confusing. I'll update it.